src/Api/ApiRequest.php

```diff
@@ -59,10 +69,14 @@ public function post($url, $params = [])
 
             return json_decode($response->raw_body);
         } catch (Exception $exception) {
-            $logs = new SaferPayLog();
-            $logs->message = $exception->getMessage() ?: "missing response";
-            $logs->payload = json_encode($params);
-            $logs->add();
+            $this->logger->error(sprintf('%s - POST request failed', self::FILE_NAME), [
```

Comment:
catch block you can use this one and add additional parameters
            $logger->error('Failed POST request', [
                'context' => [
                    'id_order' => $orderId,
                ],
                'exceptions' => ExceptionUtility::getExceptions($exception),
            ]);